diesel_cli/src/print_schema.rs

```diff
@@ -67,6 +75,14 @@ impl Display for TableDefinitions {
             }
             writeln!(f, "{}", TableDefinition(table))?;
         }
+
+        if !self.1.is_empty() {
+            write!(f, "\n")?;
+        }
+
+        for foreign_key in &self.1 {
+            write!(f, "{}", Joinable(foreign_key))?;
```

Comment:
I'd use `writeln!` here and `write!` in the `<Joinable<'a> as Display>::fmt` method